app/src/main/java/com/github/se/cyrcle/ui/report/ReviewReportScreen.kt

```diff
@@ -70,18 +70,16 @@ fun ReviewReportScreen(
   val context = LocalContext.current
 
   fun onSubmit() {
-    if (reviewId != null) {
-      val report =
-          ReviewReport(
-              uid = reviewViewModel.getNewUid(),
-              reason = selectedReason.value,
-              userId = userId ?: "",
-              review = reviewId,
-              description = reportDescription.value)
-      reviewViewModel.addReport(report, userViewModel.currentUser.value!!)
-      Toast.makeText(context, "Report added", Toast.LENGTH_SHORT).show()
-      navigationActions.goBack()
-    }
+    val report =
+        ReviewReport(
+            uid = reviewViewModel.getNewUid(),
+            reason = selectedReason.value,
+            userId = userId,
+            review = reviewId!!,
+            description = reportDescription.value)
+    reviewViewModel.addReport(report, userViewModel.currentUser.value!!)
+    Toast.makeText(context, "Report added", Toast.LENGTH_SHORT).show()
```

Comment:
Put that toast message in string.xml as usual

Reply:
Ahhh I missed one!